config.yaml

```diff
@@ -13,26 +13,26 @@
 # limitations under the License.
 ---
 categories:
-  - name: Automation
-  - name: Build Tools
-  - name: CLI
-  - name: Cloud
-  - name: Code Quality
-  - name: Continuous Integration
-  - name: Deployment
-  - name: Developer Tools
-  - name: Image Build
-  - name: Integration & Delivery
-  - name: Git
-  - name: Kubernetes
-  - name: Messaging
-  - name: Monitoring
-  - name: Networking
-  - name: Openshift
-  - name: Publishing
-  - name: Security
-  - name: Storage
-  - name: Testing
+  - Automation
```

Comment:
Shall we remove this file as we are adding all the data in the config map ?

Reply:
this is required to get configuration data in the local development , if we want to read configuration data from config map as well then I will remove it , wdyt? cc @piyush-garg @vinamra28

Reply:
I think we should keep it then for the local development atleast for now